demos/vc_issuer/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# Try to run from the script location
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+cd "$SCRIPT_DIR"
```

Comment:
```suggestion

# Make sure we always run from the issuer root
VC_ISSUER_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
cd "$VC_ISSUER_DIR/.."
```

Rewords to clarify (we're not just trying, we're effectively running from the script location) and rename (guessing `SCRIPT_DIR` was copy pasted)